src/app/container/register-tool/register-tool.component.html

```diff
@@ -359,6 +367,10 @@ <h4 mat-dialog-title>Register Tool</h4>
         novalidate
       >
         <div class="modal-body">
+          <p>
+            Create a tool with all descriptor files stored and edited on Dockstore.org. Docker images are stored on sites like Quay.io and
+            DockerHub.
```

Comment:
The images can be stored in any Docker image repo, can't they?

Reply:
Yes they can, I was just trying to list two as examples. Though if that isn't obvious I can try and change the wording.

> Create a tool with all descriptor files stored and edited on Dockstore.org. Docker images are stored on sites **_such as_** Quay.io and DockerHub

Reply:
Well not any Docker image repo, there are a select few we support (though easyish to add more).

Reply:
Whoops, the way it was wrapping the right side was clipped, and I only saw 

```
...Docker images are stored on 
Dockerhub
```
And totally didn't see `are stored on sites like Quay.io and`. So it was and is good. Sorry for the confusion.